frontend/src/components/phones/onboarding/RealPhoneSetup.tsx

```diff
@@ -0,0 +1,318 @@
+import { Localized, useLocalization } from "@fluent/react";
+import styles from "./RealPhoneSetup.module.scss";
+import PhoneVerify from "./images/phone-verify.svg";
+import EnteryVerifyCode from "./images/enter-verify-code.svg";
+import EnteryVerifyCodeError from "./images/verify-code-error.svg";
```

Comment:
Should these be `EnterVerifyCode*` instead of `Entery` ?

Reply:
Updated in de725280b1a1fa9cb499491b2078e30f3e43dcd1